src/azure-cli/azure/cli/command_modules/containerapp/_params.py

```diff
@@ -202,10 +202,17 @@ def load_arguments(self, _):
         c.argument('certificate_password', options_list=['--password', '-p'], help='The certificate file password')
         c.argument('prompt', options_list=['--show-prompt'], action='store_true', help='Show prompt to upload an existing certificate.')
 
+    with self.argument_context('containerapp env certificate create') as c:
+        c.argument('hostname', options_list=['--hostname'], help='The custom domain name.')
+        c.argument('certificate_name', options_list=['--certificate-name', '-c'], help='Name of the managed certificate which should be unique within the Container Apps environment.')
+        c.argument('validation_method', options_list=['--validation-method', '-v'], help='Validation method of custom domain ownership. Supported methods are HTTP, CNAME and TXT.')
```

Comment:
If the param is enumeable, please use `arg_type=get_enum_type()` to define the param. After defining with that, it is not necessary to specify the available values and validate the users' input.
```suggestion
        c.argument('validation_method', arg_type=get_enum_type(['HTTP', 'CNAME', 'TXT']), options_list=['--validation-method', '-v'], help='Validation method of custom domain ownership.')
```